arch/AArch64/AArch64Disassembler.c

```diff
@@ -271,1133 +311,1213 @@ bool AArch64_getInstruction(csh ud, const uint8_t *code, size_t code_len,
-	// S<op0>_<op1>_<Cn>_<Cm>_<op2>, so decoding system registers always succeeds.
-	return Success;
+  // Every system register in the encoding space is valid with the syntax
+  // S<op0>_<op1>_<Cn>_<Cm>_<op2>, so decoding system registers always succeeds.
+  return Success;
 }
 
 static DecodeStatus DecodeMSRSystemRegister(MCInst *Inst, unsigned Imm,
-		uint64_t Address,
-		const void *Decoder)
-{
-	MCOperand_CreateImm0(Inst, Imm);
+                                            uint64_t Address,
+                                            const void *Decoder) {
+  MCOperand_CreateImm0(Inst, Imm);
 
-	return Success;
+  return Success;
 }
 
 static DecodeStatus DecodeFMOVLaneInstruction(MCInst *Inst, unsigned Insn,
-		uint64_t Address,
-		const void *Decoder)
-{
-	// This decoder exists to add the dummy Lane operand to the MCInst, which must
-	// be 1 in assembly but has no other real manifestation.
-	unsigned Rd = fieldFromInstruction(Insn, 0, 5);
-	unsigned Rn = fieldFromInstruction(Insn, 5, 5);
-	unsigned IsToVec = fieldFromInstruction(Insn, 16, 1);
-
-	if (IsToVec) {
-		DecodeFPR128RegisterClass(Inst, Rd, Address, Decoder);
-		DecodeGPR64RegisterClass(Inst, Rn, Address, Decoder);
-	} else {
-		DecodeGPR64RegisterClass(Inst, Rd, Address, Decoder);
-		DecodeFPR128RegisterClass(Inst, Rn, Address, Decoder);
-	}
+                                              uint64_t Address,
+                                              const void *Decoder) {
+  // This decoder exists to add the dummy Lane operand to the MCInst, which must
+  // be 1 in assembly but has no other real manifestation.
+  unsigned Rd = fieldFromInstruction(Insn, 0, 5);
+  unsigned Rn = fieldFromInstruction(Insn, 5, 5);
+  unsigned IsToVec = fieldFromInstruction(Insn, 16, 1);
+
+  if (IsToVec) {
+    DecodeFPR128RegisterClass(Inst, Rd, Address, Decoder);
+    DecodeGPR64RegisterClass(Inst, Rn, Address, Decoder);
+  } else {
+    DecodeGPR64RegisterClass(Inst, Rd, Address, Decoder);
+    DecodeFPR128RegisterClass(Inst, Rn, Address, Decoder);
+  }
 
-	// Add the lane
-	MCOperand_CreateImm0(Inst, 1);
+  // Add the lane
+  MCOperand_CreateImm0(Inst, 1);
 
-	return Success;
+  return Success;
 }
 
 static DecodeStatus DecodeVecShiftRImm(MCInst *Inst, unsigned Imm,
-		unsigned Add)
-{
-	MCOperand_CreateImm0(Inst, Add - Imm);
-	return Success;
+                                       unsigned Add) {
+  MCOperand_CreateImm0(Inst, Add - Imm);
+  return Success;
 }
 
 static DecodeStatus DecodeVecShiftLImm(MCInst *Inst, unsigned Imm,
-		unsigned Add)
-{
-	MCOperand_CreateImm0(Inst, (Imm + Add) & (Add - 1));
-	return Success;
+                                       unsigned Add) {
+  MCOperand_CreateImm0(Inst, (Imm + Add) & (Add - 1));
+  return Success;
 }
 
 static DecodeStatus DecodeVecShiftR64Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder)
-{
-	return DecodeVecShiftRImm(Inst, Imm, 64);
+                                         uint64_t Addr, const void *Decoder) {
```

Comment:
many places, like this, does not really change anything, but indentation. please make sure the commit only include important changes.

Reply:
I basically copied from llvm and changed C++ to C.  Are you sure you don't want to use llvm's formatting it would make updates in the future easier?

Reply:
we use quite different coding style, which is similar to Linux kernel coding style: https://www.kernel.org/doc/html/v4.11/process/coding-style.html

and we use tabs for indentation.